src/shared-styles/_rem.scss

```diff
@@ -1,9 +1,14 @@
 @use "font";
 @use "vendor/sass-rem/rem" as sass-rem with (
   $rem-baseline: font.$size,
-  $rem-fallback: false,
 );
 
+@forward "vendor/sass-rem/rem";
+
 @function calculate($values...) {
   @return sass-rem.rem($values...);
 }
+
+@mixin calculate($properties, $values...) {
```

Comment:
Same name for mixin and function, matching the approach of the underlying library.